libs/ui-watt/src/lib/components/table/watt-table.component.scss

```diff
@@ -23,6 +23,7 @@ watt-table .mat-table {
   grid-auto-rows: max-content;
   height: 100%;
   overflow: auto;
+  isolation: isolate;
```

Comment:
Praise: Cool, I didn't know this property

Reply:
Neither did I 😄 Much better than the old hacks to force a new stacking context!